driver-core/src/main/java/com/datastax/driver/core/TableOptionsMetadata.java

```diff
@@ -206,6 +218,19 @@ public String getComment() {
    * @return the read repair chance set for table (in [0.0, 1.0]).
    */
   public double getReadRepairChance() {
+    return readRepairChance;
+  }
+
+  /**
+   * Returns the read_repair option for this table. <b>NOTE:</b> this is a Cassandra® 4.0 and newer
+   * option (described here<a
+   * href="http://cassandra.apache.org/doc/latest/operating/read_repair.html">here:
+   * http://cassandra.apache.org/doc/latest/operating/read_repair.html</a>). Possible values are
```

Comment:
The link is included in the link's text, is that on purpose?

Reply:
Yes, my intent was to link to the Cassandra documentation, and have the link show the full address.